docs/kyma/docs/05-01-overview.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Overview
+type: Configuration
+---
+
+## Default settings
+
+During the installation process, the Installer applies onto a cluster or Minikube all components defined in the `installer-cr-cluster.yaml.tpl` and `installer-cr.yaml.tpl` templates with their configuration defined in `values.yaml` files. It also imports the configuration settings defined in the `installer-config-cluster.yaml.tpl` and `installer-config-local.yaml.tpl` templates located under the `installation/resources` subfolder.
```

Comment:
`installer-cr-cluster.yaml.tpl` or `installer-cr.yaml.tpl`

`imports the configuration settings` = `imports the configuration overrides`?

Reply:
Changed that :)